framework/console/controllers/FixtureController.php

```diff
@@ -431,11 +437,18 @@ private function getFixturesConfig($fixtures)
         $config = [];
 
         foreach ($fixtures as $fixture) {
-            $isNamespaced = (strpos($fixture, '\\') !== false);
-            $fullClassName = $isNamespaced ? $fixture . 'Fixture' : $this->namespace . '\\' . $fixture . 'Fixture';
+            // $isNamespaced = (strpos($fixture, '\\') !== false);
+            // this would have incorrect result on windows-based systems in case of fixtures placed in subdirectories
 
-            if (class_exists($fullClassName)) {
-                $config[] = $fullClassName;
+            // istead of guessing if $fixture is namespaced, let's just check it _both_ as non-namespaced and namespaced
+            $className = $fixture . 'Fixture';
+            if (class_exists($className)) {
```

Comment:
If both namespaced and non-namespaced fixtures with the same base name exist there will be conflict.